src/clib/lib/job_queue/torque_driver.cpp

```diff
@@ -530,9 +535,9 @@ torque_driver_get_qstat_status(torque_driver_type *driver,
     job_status_type status = JOB_QUEUE_STATUS_FAILURE;
 
     {
-        const char **argv = (const char **)util_calloc(2, sizeof *argv);
-        argv[0] = driver->qstat_opts;
-        argv[1] = jobnr_char;
+        char const *hard_coded_qstat_option = "-f";
```

Comment:
Can this variable name be more explicit? ex. `full_output_option`?

Reply:
I think it is valuable to tag it in the code that this is an option that the driver hard-codes. But a comment explaining what `-f` means is absolutely warranted!

Reply:
Comment added.